lochness/config/__init__.py

```diff
@@ -4,20 +4,34 @@
 import yaml.reader
 import getpass as gp
 import cryptease as crypt
+import string
 
 logger = logging.getLogger(__name__)
 
+
 def load(f, archive_base=None):
     '''load configuration file and keyring'''
     logger.debug('loading configuration')
+
     with open(os.path.expanduser(f), 'rb') as fp:
         Lochness = _read_config_file(fp)
+
     if archive_base:
         Lochness['phoenix_root'] = archive_base
     if 'phoenix_root' not in Lochness:
-        raise ConfigError('need either --archive-base or \'phoenix_root\' in config file')
+        raise ConfigError('need either --archive-base or '
+                          '\'phoenix_root\' in config file')
     Lochness['phoenix_root'] = os.path.expanduser(Lochness['phoenix_root'])
     Lochness['keyring_file'] = os.path.expanduser(Lochness['keyring_file'])
+
+    # box file pattern strings from the config to string template
+    # regardless of the selected study in the args
+    for _, study_dict in Lochness['box'].items():
+        for _, modality_values in study_dict['file patterns'].items():
+            for modality_dict in modality_values:
+                modality_dict['pattern'] = \
+                    string.Template(modality_dict['pattern'])
```

Comment:
Why is this in a new line and not in the line before?

Reply:
The line was too long (84>79)

We could break the line after `string.Template`, but I thought putting the whole line in the new line reads better. What do you think?

Reply:
I thought PEP convention is 120 characters but just noticed it is 80 characters. So disregard my comment.